about.html

```diff
@@ -0,0 +1,113 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Lato:wght@400;700;900&family=Poppins:ital,wght@0,200;0,400;0,600;1,500&display=swap" rel="stylesheet">
+<link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="about.css">
+  <title>Concert Page</title>
+</head>
+<body>
+  <section class="introduction">
+    <div id="overlay">
+      <div id="close-container" onclick="menuClose()">
+        <a id="closer" onclick="menuClose()">
+          <img src="./images/Icon.png" alt="Close" onclick="menuClose()">
+        </a>
+      </div>
+      <div id="menucontainer">
+          <a href="" onclick="menuClose()">Home</a>
+          <a href="" onclick="menuClose()">About</a>
+          <a href="" onclick="menuClose()">Program</a>
+          <a href="" onclick="menuClose()">Join</a>
+          <a href="" onclick="menuClose()">News</a>
+      </div>
+    </div>
+  <header class="primary-header">
+    <ul id="social-icons">
+      <li class="media-icons"><img src="./images/facebook.png" alt=""></li>
+      <li class="media-icons"><img src="./images/twitter.png" alt=""></li>
+      <li class="media-icons"><img src="./images/instagram.png" alt=""></li>
+      <li class="media-icons"><img src="./images/linkedin.png" alt=""></li>
+    </ul>
+  </header>
+    <header class="toolbar">
+      <img src="./images/summer.png" alt="" class="summer_img" >
+      <div >
+        <img src='./images/mobile_menu.png' alt="mobile_menu" id="toolbar-img" onclick= menuOpen() >
+      </div>
+      <ul id="link">
+        <li class="links"><a href="index.html">Home</a></li>
+        <li class="links"><a href="">About</a></li>
+        <li class="links"><a href="">Program</a></li>
+        <li class="links"><a href="">Join</a></li>
+        <li class="links"><a href="">News</a></li>
+      </ul>
+    </header>
+    <h1>Hello sharing world!</h1>
+    <h2>CREATIVE SUMMER CONCERT 2020</h2>
+    <P class="primary-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+      nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in 
+      reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+      Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt 
+      mollit anim id est laborum.</P>
+      <p>Please contact us per Email for any further questions about Summer concert series</p>
+      <strong id="subscribe">summerconcert@hotmail.com</strong>
+  </section>
+  <section class="middle">
```

Comment:
- [ ] For the about page there is an empty space on the right side, it could be because one of the elements is overflowing. Kindly fix the issue 💯

 
![Animation5](https://user-images.githubusercontent.com/50721479/190721038-431f7fbb-a14a-40ca-a39f-cb4311c3bb87.gif)